tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,16 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
-
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
```

Comment:
We'd also want to explicitly test to see if the inventory we worked on was modified, this makes our test more robust. 
```suggestion
    assert result == False
    assert "a" in vendor.inventory
    assert "b" in vendor.inventory
    assert "c" in vendor.inventory
    assert item not in vendor.inventory
```

Reply:
Thanks!